tests/snuba/api/endpoints/test_organization_events_v2.py

```diff
@@ -897,6 +898,82 @@ def test_count_miserable_alias_field(self):
         data = response.data["data"]
         assert data[0]["count_miserable_user_300"] == 2
 
+    def test_count_miserable_new_alias_field(self):
+        project = self.create_project()
+
+        ProjectTransactionThreshold.objects.create(
+            project=project,
+            organization=project.organization,
+            threshold=400,
+            metric=TransactionMetric.DURATION.value,
+        )
+
+        events = [
+            ("one", 400),
+            ("one", 400),
+            ("two", 3000),
+            ("two", 3000),
+            ("three", 300),
+            ("three", 3000),
+        ]
+        for idx, event in enumerate(events):
+            data = load_data(
+                "transaction",
+                timestamp=before_now(minutes=(1 + idx)),
+                start_timestamp=before_now(minutes=(1 + idx), milliseconds=event[1]),
+            )
+            data["event_id"] = f"{idx}" * 32
+            data["transaction"] = f"/count_miserable/horribilis/{event[0]}"
+            data["user"] = {"email": f"{idx}@example.com"}
+            self.store_event(data, project_id=project.id)
+
+        query = {
+            "field": [
+                "transaction",
+                "project_threshold_config",
```

Comment:
I'm tempted to say that this column should be hidden from the user.

As a user, I don't want to have to add a column for user misery to work. I should be able to add user misery, and it'll automatically insert this config column behind the scenes.